pom.xml

```diff
@@ -5,12 +5,12 @@
     <modelVersion>4.0.0</modelVersion>
 
     <groupId>org.jboss.bom.rhba</groupId>
```

Comment:
RHBA = Red Hat Business Automation, formerly BPMS, so I think we probably need to change this, afaik it was Business Automation Suite for the whole set, but this may have been changed, truth be told I don't see this term being used much beyond Prakash's original presentation when the renaming was to take place.

Reply:
Prakash sent a mail just now, Business Automation is the suite name, so we're good here.

Reply:
that is strictly wrong, as we have for RHDM org.jboss.bom.rhdm, this should follow the same logic I think.

Reply:
@mareknovotny @AntStephenson. Thank you for your comments. Good point, we should be consistent with RHDM. Let me send new PR that will fix this.